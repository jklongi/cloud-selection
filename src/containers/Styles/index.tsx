import { ThemeProvider } from '@material-ui/core/styles'
import CssBaseline from '@material-ui/core/CssBaseline'

import theme from 'styles/themes'

type Props = {
    children: React.ReactNode
}

const Styles = ({ children }: Props) => (
    <ThemeProvider theme={theme}>
        <CssBaseline />
        {children}
    </ThemeProvider>
)

export default Styles
