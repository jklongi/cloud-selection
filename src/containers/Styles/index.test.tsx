import { mount } from 'enzyme'
import Styles from './index'

describe('<Styles />', () => {
    it('matches snapshot', () => {
        const wrapper = mount(
            <Styles>
                <div>Hello</div>
            </Styles>,
        )
        expect(wrapper).toMatchSnapshot()
    })
    it('renders its children', () => {
        const wrapper = mount(
            <Styles>
                <div>Hello</div>
            </Styles>,
        )
        expect(wrapper.find('div').text()).toEqual('Hello')
        expect(wrapper.find('div')).toHaveLength(1)
    })
})
