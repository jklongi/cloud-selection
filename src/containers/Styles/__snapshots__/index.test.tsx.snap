// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Styles /> matches snapshot 1`] = `ReactWrapper {}`;
